source/jormungandr/tests/ptref_tests.py

```diff
@@ -225,7 +225,7 @@ def test_dataset_by_contributor(self):
 
     def test_line(self):
         """test line formating"""
-        response = self.query_region("v1/lines")
```

Comment:
I'd rather have a dedicated test just testing that part (copy paste of the beginning of this one), but...